src/sentry/__init__.py

```diff
@@ -48,6 +48,14 @@ def get_version():
         return '%s.%s' % (__version__, __build__)
     return __version__
 
+
+def is_docker():
+    # One of these environment variables are guaranteed to exist
+    # from our official docker images.
+    # SENTRY_VERSION is from a tagged release, and SENTRY_BUILD is from a
+    # a git based image.
+    return 'SENTRY_VERSION' in os.environ or 'SENTRY_BUILD' in os.environ
```

Comment:
I think it would be useful to collect the actual version of Docker being ran. `DOCKER_VERSION` is a standard env var in all the containers it runs. Could be something like:
`return os.environ['DOCKER_VERSION'] if 'DOCKER_VERSION' in os.environ else '-1'`